### PR TITLE
button-view-analytic added

### LINKED ISSUE #2925: Button "View Analytic" on an API Profile
**Title**: 

Add the button "View Analytic" on an API Profile page

**Description**:

The button will redirect API manager from API profile directly to current API Analytic page

### PR DESCRIPTION
**Pull request by krashna-deligence**
Closes #2925 

## Changes
Describe your changes here as a bulleted list:
- Add button and redirect to analytics

## Developer checklist
This checklist is to be completed by the PR developer:

- [ ] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [ ] This pull request does not decrease project test coverage
- [ ] If the code changes existing database collection(s), migration has been written
- [ ] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @username1

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

File: apinf_packages/apis/client/profile/header/header.html [hunk 1]
```diff
@@ -37,6 +37,13 @@ <h1 id="api-header">
 
           {{> apiBackendRating api }}
         </div>
+        <div>
+          {{# if proxyBackend }}
+            <!-- View Analytics -->
+            <a href="{{ pathFor 'apiAnalyticPage' apiSlug=api.slug }}" class="btn btn-primary sm">View Analytics</a>
```
Comment (marla-singer) [inline apinf_packages/apis/client/profile/header/header.html]: 1. Nested elements has to have a new owned line
2. What does class "sm" mean? We've used bootsrtap-3. Remove them or change to btn-sm if it needs
3. After fixing changes, squash the second commit to the first using `git rebase develop -i`

File: apinf_packages/apis/client/profile/header/header.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ import { Template } from 'meteor/templating';
 // Collection imports
 import Apis from '/apinf_packages/apis/collection';
 
+// Collection imports
+import ProxyBackends from '/apinf_packages/proxy_backends/collection';
```
Comment (marla-singer) [inline apinf_packages/apis/client/profile/header/header.js]: Travis fails with this line. Remove it

File: apinf_packages/apis/client/profile/header/header.html [hunk 3]
```diff
@@ -37,6 +37,13 @@ <h1 id="api-header">
...
+
```
Comment (marla-singer) [inline apinf_packages/apis/client/profile/header/header.html]: Remove the empty line

**Comment by krashna-deligence** [comment]
@marla-singer Travis Ci failed the check of code on testing please review below screenshot.
screenshot from 2017-12-08 12-30-05

**Comment by marla-singer** [comment]
@krashna-deligence Yeah, I've seen it. I don't know reasons but your code doesn't relate to it.
Merge it anyway

**Pull request merged**